src/code/z_play.c

```diff
@@ -3,6 +3,9 @@
 #include "z64shrink_window.h"
 #include "z64view.h"
 
+extern Struct_80140E80 D_801F6D38;
```

Comment:
Would be nice to brainstorm a name for the struct at least. It seems it should have `bg` involved somehow, but beyond that I'm not sure.